src/app/components/core/dashboard/dashboard.component.ts

```diff
@@ -5,6 +5,7 @@ import { takeWhile, takeUntil } from 'rxjs/operators';
 
 import { AlertService, PingService, StatisticsService } from '../../../services';
 import { GRAPH_REFRESH_INTERVAL, STATS_HISTORY_TIME_FILTER } from '../../../utils';
+import { analyze } from 'tern';
```

Comment:
from where we got tern package in?

Reply:
Mistakenly and useless import, removing it..